src/python/foglamp/core/routes.py

```diff
@@ -16,12 +16,30 @@
 def setup(app):
     # app.router.add_route('POST', '/foglamp/a-post-req', api.a_post_req, expect_handler = aiohttp.web.Request.json)
     app.router.add_route('GET', '/foglamp/ping', api.ping)
+
+    # Configuration
     app.router.add_route('GET', '/foglamp/categories', api.get_categories)
     app.router.add_route('GET', '/foglamp/category/{category_name}', api.get_category)
     app.router.add_route('GET', '/foglamp/category/{category_name}/{config_item}', api.get_category_item)
     app.router.add_route('PUT', '/foglamp/category/{category_name}/{config_item}/{value}', api.set_configuration_item)
```

Comment:
This is not as per the specification. In the specification the value is in the payload, not a parameter in the URL